src/api/messages/messagesFlags.js

```diff
@@ -1,6 +1,16 @@
 /* @flow strict-local */
-import type { ApiResponse, Auth } from '../transportTypes';
+import type { ApiResponseSuccess, Auth } from '../transportTypes';
 import { apiPost } from '../apiFetch';
 
-export default (auth: Auth, messages: number[], op: string, flag: string): Promise<ApiResponse> =>
+export type ApiResponseMessagesFlags = {|
```

Comment:
```
messagesFlags: Return list of messages from API response.

The API gives us a list of messages we successfully marked as read,
which is useful to expose to the rest of our code.
```

Commit message nit: Not just marked-as-read messages, I think; the [doc](https://zulip.com/api/update-message-flags) says "An array with the IDs of the modified messages." (So, e.g., this includes changing the starred/unstarred status.)

Reply:
Done.